### PR TITLE
fix for conflict with listonic library

### PR DESCRIPTION
**Pull request by gczudowski**
### Checklist
- [ ] [Reported an issue](http://cordova.apache.org/contribute/issues.html) in the JIRA database
- [ ] Commit message follows the format: "CB-3232: (android) Fix bug with resolving file paths", where CB-xxxx is the JIRA ID & "android" is the platform affected.
- [ ] Added automated test coverage as appropriate for this change.

### CODE REVIEW

**Review by robertgacki — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,4 @@ Thumbs.db
 *.user
 
 /node_modules/**
+.idea
```
Comment (KamilKujawa) [inline .gitignore:16]: There is no line break at the end of the file :D

**Review by KamilKujawa — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (ziggurad) [reply .gitignore:16]: I agree!

Comment (gilek) [reply .gitignore:16]: Yeah, critical issue

Comment (gczudowski) [reply .gitignore:16]: co słychać chłopaki? :D

**Review by KamilKujawa — APPROVED** [review]

**Pull request merged**